src/plugins/intel_gpu/tests/unit/test_cases/eltwise_gpu_test.cpp

```diff
@@ -4705,12 +4705,17 @@ struct eltwise_layout_test_params {
 #define CASE_ELTWISE_TEST1  eltwise_mode::sum, {1, 2, 1, 1}, {4, 2, 4, 4}, format::b_fs_yx_fsv16, format::bfyx, "generic_eltwise_ref"
 #define CASE_ELTWISE_TEST2  eltwise_mode::sum, {4, 1, 4, 4}, {1, 5, 1, 1}, format::b_fs_yx_fsv16, format::bfyx, "generic_eltwise_ref"
 #define CASE_ELTWISE_TEST3  eltwise_mode::sum, {4, 5, 4, 1}, {4, 1, 4, 1}, format::b_fs_yx_fsv16, format::bfyx, "generic_eltwise_ref"
-#define CASE_ELTWISE_TEST4  eltwise_mode::sum, {4, 2, 4, 4}, {1, 1, 1, 1}, format::b_fs_yx_fsv16, format::bfyx, "eltwise_blocked_opt"
+#define CASE_ELTWISE_TEST4  eltwise_mode::sum, {4, 2, 4, 4}, {1, 1, 1, 1}, format::b_fs_yx_fsv16, format::bfyx, "generic_eltwise_ref"
```

Comment:
Please ensure this test works with the blocked_opt kernel, otherwise we may face performance regressions

Reply:
By adding the vec_size check for bfyx type, based on this {1,1,1,1} shape, the logicalSize is =1.  So, the vec_size check will fail the validation.  I think this test shouldn't use eltwise_blocked_opt.

Reply:
But it seems we just have to additionally allow bfyx fromat with LogicalSize == 1, isn't it? As it worked previously like that, we can't change this behavior, because it might change the kernel for some real model and cause significant performance drop

Reply:
Can't this case be handled with the following code?
```
            bool is_valid = input.LogicalSize() == 1;          // Scalar value broadcast
            is_valid |= input.LogicalSize() % vec_size == 0 && // Feature values broadcast
                        input.LogicalSize() == input.Feature().v &&
                        input.LogicalSize() == output.Feature().v &&
                        GetInnerBatchBlockSize(input) == 1;
            if (!is_valid) {
                return false;
            }
```

Reply:
thanks.  Update the change.